test/arrays/sparse/copyRemoteSparse-cs.chpl

```diff
@@ -0,0 +1,64 @@
+use CommDiagnostics, LayoutCS;
+
+config param compressRows = true;
+
+config const countComms = true;
+
+const D = {1..9, 1..9};
+
+var SD: sparse subdomain(D) dmapped new dmap(new CS(compressRows)) = [i in 1..9] (i,i);
+
+var A: [SD] real;
+
+for i in 1..9 do
+  A[i,i] = i + i/10.0;
+
+on Locales[numLocales-1] {
```

Comment:
`on Locales.last` ? :)

Reply:
In my defense, I wrote this test before that conversation yesterday!  :D  

The one slight hesitation I have about retraining myself is how long it will take me to remember whether or not .last takes parens or not.  :(